### PR TITLE
PP-7445 URL Restructure > Feedback form

### PR DESCRIPTION
**Pull request by iqbalgds**
- Update URL structure for `feedback` routes so it no longer uses `service`
  or `gateway account` in the URL.
- Update routes.js
  - feedback routes - remove middleware for getting the gateway account or service.
- Update the Feedback NJK file
  - Add a `back to my services` link
- Update functional tests
  - Rename files to make it clear they are functional tests.
  - Remove the stubs for `service` and `gateway account` details.
- Add Cypress test for the feedback form.

image

### CODE REVIEW

File: app/controllers/feedback/post-index.controller.js [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
+  const message = `Feedback rating: ${req.body['feedback-rating']}
 ----
 ${req.body['feedback-suggestion']}`
 
   const opts = {
     email: req.body.email,
     name: '(no name supplied)',
     type: 'question',
-    subject: `Feedback from: ${req.body['service-name']}`,
+    subject: `General feedback`,
```
Comment (kbottla) [inline app/controllers/feedback/post-index.controller.js:17]: Shall we say we `Feedback from service` to make it clear for support?

File: app/views/feedback/index.njk [hunk 2]
```diff
@@ -4,6 +4,16 @@
 Give feedback — GOV.UK Pay
 {% endblock %}
 
+{% block beforeContent %}
+  {{ super() }}
+  {{
+    govukBackLink({
+      text: "Back to my services",
```
Comment (kbottla) [inline app/views/feedback/index.njk:11]: Can we change this to "My services" as users can visit page from any page

**Review by kbottla — COMMENTED** [review]
Changes looks good to me. Few minor suggestions on naming

File: app/views/feedback/index.njk (hunk 2 above)
Comment (iqbalgds) [reply app/views/feedback/index.njk:11]: Good point

**Review by kbottla — APPROVED** [review]

File: app/controllers/feedback/post-index.controller.js (hunk 1 above)
Comment (iqbalgds) [reply app/controllers/feedback/post-index.controller.js]: Done

**Review by kbottla — APPROVED** [review]

**Pull request merged**